examples/bri-3/src/bri/state/agents/state.agent.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@nestjs/common';
+
+import MerkleTree from 'merkletreejs';
+import { BpiAccountStorageAgent } from '../../identity/bpiAccounts/agents/bpiAccountsStorage.agent';
+import { BpiAccount } from '../../identity/bpiAccounts/models/bpiAccount';
+import { MerkleTreeAgent } from '../../merkleTree/agents/merkleTree.agent';
+import { MerkleTreeStorageAgent } from '../../merkleTree/agents/merkleTreeStorage.agent';
+import { Witness } from '../../zeroKnowledgeProof/models/witness';
+
+// TODO: We should follow this approach everywhere for storage
```

Comment:
TODOs are open issues or will be resolved before PR merged?

Reply:
@Therecanbeonlyone1969  This code was copied from Ognjen's PR. This question has been addressed in PR #732